sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/generators.scala

```diff
@@ -204,6 +204,17 @@ case class Stack(children: Seq[Expression]) extends Generator {
   }
 }
 
+case class GeneratorOuter(child: Generator) extends UnaryExpression
+    with Generator {
```

Comment:
Couldn't make it work with Unevaluable because Generator.eval is not the same as Unevaluable.eval.